farcy/__init__.py

```diff
@@ -286,14 +286,16 @@ def get_issues(self, pfile):
         retval = {}
 
         try:
-            tmpdir = mkdtemp()
+            path_in_repo = os.path.dirname(pfile.filename)
+            os.makedirs(path_in_repo)
+            tmpdir = mkdtemp(dir = path_in_repo)
```

Comment:
Do we want this the other way around - the temp directory first then the repo's directory structure after?

Reply:
Please make the `tmpdir` first so that this whole directory structure lives in /tmp/.